staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -156,6 +156,9 @@ type VolumeSource struct {
 	// CSI (Container Storage Interface) represents ephemeral storage that is handled by certain external CSI drivers (Beta feature).
 	// +optional
 	CSI *CSIVolumeSource `json:"csi,omitempty" protobuf:"bytes,28,opt,name=csi"`
+	// Ephemeral represents ephemeral storage that is handled by a normal storage driver (Alpha feature).
```

Comment:
More description!

Probably describe when you would use this and which volumes would be supported.  Likewise describing the connection between this and pvc in the same struct (and improving that description).  `Use X when A, and Y when B`